code/modules/spells/ability_types/realized_aimed.dm

```diff
@@ -131,7 +133,7 @@
 	if(ishostile(target))
 		var/mob/living/simple_animal/hostile/H = target
 		H.TemporarySpeedChange(1, 10 SECONDS)
-		damage = max(0.02*H.maxHealth, 40)
+		damage = max(0.02*H.maxHealth, 150)
```

Comment:
Issue with this; If you wanted to make it more effective increase the %. The only things I *know of* that would not hit this minimum is the Arbiter, White Night (15k god damn), and The Claw (barely). Most Alephs have about 4k hp, so a +1% to this would make it go from 80 damage to 120. Simply increasing the minimum damage kinda defeats the purpose of it being % damage. Hell it's still PALE damage so you could justify making it 4 or 5% honestly.